doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -37,6 +37,105 @@ It is recommended to follow the “url” link values,
 applicable to retrieve resources. Instead of constructing your own URLs,
 to keep your client decoupled from implementation details.
 
+## JSON entity representation
+
+Many operations use the same JSON entity representation format which is described in the present section. Each
```

Comment:
Regarding: 

> Property values can be single values or they can include, type and metadata attributes as well

It think it would be more precise the following working: 

"Property values can be single values or they can include, `type` and properties representing metadata as well".

Regarding: 

> type : A JSON string which represents the attribute type provided by the entity creator

May be changed to:

type : A JSON string which represents the attribute type provided by the entity creator (or was implicitely set if the creator doesn't specified type at creation date).

Or the comment about "or was implicitely set if the creator doesn't specified type at creation date" could be included in some other part of the documentation (maybe better, to keep this part simpler).